src/printer.cpp

```diff
@@ -148,11 +150,19 @@ std::string Printer::printComponent(ComponentPtr component) const
         repr += " id=\"" + component->getId() + "\"";
     }
     size_t variable_count = component->variableCount();
-    if ((variable_count > 0) || (component->getMath().length())) {
+    size_t reset_count = component->resetCount();
```

Comment:
Minor: for consistency, rename `variable_count` to `variableCount` and `reset_count` to `resetCount`.